### ISSUE TITLE
Error: SwiftSyntax parser library isn't compatible

### ISSUE THREAD

**Issue by Panajev** [SwiftDocOrg/swift-doc#186]
Hello @mattt ,

I am trying to use the source code installation method and I get the error provided as the Issue title when running the following commands building up the tool and generating docs and coverage maps:
```bash
        swift run swift-doc generate "${SOURCE_CODE}" --module-name <MODULE_NAME> --output "${REPO_DOCS}"
        swift run swift-doc coverage "${SOURCE_CODE}" --output "${REPO_DOCS}/docCoverage.json"
        swift run swift-doc diagram "${SOURCE_CODE}" > "${REPO_DOCS}/graph.dot"
```

**Output - CI (Catalina, Xcode 12)**:
```bash
ld: warning: dylib (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/lib_InternalSwiftSyntaxParser.dylib) was built for newer macOS version (10.12) than being linked (10.10)
[310/310] Linking swift-doc

Error: SwiftSyntax parser library isn't compatible
```

**Output - Locally (Big Sur, Xcode 12)**:
```bash
ld: warning: dylib (/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/macosx/lib_InternalSwiftSyntaxParser.dylib) was built for newer macOS version (10.12) than being linked (10.10)
[310/310] Linking swift-doc
Error: SwiftSyntax parser library isn't compatible
```

**Comment by mattt** [comment]
Hi @Panajev, we recently merged support for Swift 5.3 with #183. Working from the latest HEAD of master, I'm unable to reproduce these problems on Catalina with Xcode 12.

```terminal
$ xcodebuild -version
Xcode 12.0
Build version 12A7209
$ swift --version
Apple Swift version 5.3 (swiftlang-1200.0.29.2 clang-1200.0.30.1)
Target: x86_64-apple-darwin19.6.0
$ swift run swift-doc generate Sources/SwiftDoc --module-name SwiftDoc --format html
```

I plan to cut a release for Swift 5.3 later today. In the meantime, please try building from the latest on master.

**Comment by mattt** [comment]
See also #165, #176

**Comment by mattt** [comment]
This is fixed in swift-doc 1.0.0-beta.5
